components/raftstore/src/store/worker/region.rs

```diff
@@ -541,9 +543,11 @@ impl<R: CasualRouter> Runner<R> {
         router: R,
     ) -> Runner<R> {
         Runner {
-            pool: ThreadPoolBuilder::with_default_factory(thd_name!("snap-generator"))
-                .thread_count(GENERATE_POOL_SIZE)
-                .build(),
+            pool: Builder::new("snap-generator")
+                    .max_thread_count(GENERATE_POOL_SIZE)
+                    .min_thread_count(0)
```

Comment:
You don't need to configure it.

Reply:
Then the following line will be left unused. 
`const GENERATE_POOL_SIZE: usize = 2;`
Is it okay to remove this line, then?

Reply:
Oh, I mean you don't need to configure `min_thread_count`, `max_thread_count` is still necessary.